edge-modules/TwinTester/Settings.cs

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace TwinTester
+{
+    using System;
+    using System.IO;
+    using Microsoft.Azure.Devices.Client;
+    using Microsoft.Extensions.Configuration;
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Converters;
+    using Newtonsoft.Json.Serialization;
+
+    [JsonObject(NamingStrategyType = typeof(CamelCaseNamingStrategy))]
+    public class Settings
+    {
+        static readonly Lazy<Settings> DefaultSettings = new Lazy<Settings>(
+            () =>
+            {
+                IConfiguration configuration = new ConfigurationBuilder()
+                    .SetBasePath(Directory.GetCurrentDirectory())
+                    .AddJsonFile("config/settings.json", optional: true)
+                    .AddEnvironmentVariables()
+                    .Build();
+
+                return new Settings(
+                    configuration.GetValue<string>("IOTEDGE_DEVICEID", string.Empty),
+                    configuration.GetValue<string>("IOTEDGE_MODULEID", string.Empty),
+                    configuration.GetValue<double>("JitterFactor", 0),
+                    configuration.GetValue<int>("TwinUpdateCharCount", 1),
+                    configuration.GetValue<TimeSpan>("TwinUpdateFrequency", TimeSpan.FromMilliseconds(500)),
+                    configuration.GetValue<TimeSpan>("TwinUpdateFailureThreshold", TimeSpan.FromMinutes(1)), // TODO: tune
+                    configuration.GetValue<TransportType>("TransportType", TransportType.Amqp_Tcp_Only),
+                    configuration.GetValue<string>("AnalyzerUrl", "http://analyzer:15000"),
+                    configuration.GetValue<string>("ServiceClientConnectionString", string.Empty),
+                    configuration.GetValue<string>("StoragePath", string.Empty),
```

Comment:
The storage path default value is empty string, which matches the analyzer, but does this make sense? I'm not sure how this is used by rocksdb.